azext_iot/common/utility.py

```diff
@@ -619,3 +621,24 @@ def is_valid_dtmi(dtmi):
     if not pattern.match(dtmi):
         return False
     return True
+
+
+@contextmanager
+def capture_stderr():
```

Comment:
Is this working as intended or actually not being used?

Reply:
I will need a refresher on what this is supposed to do

Reply:
also will need some work on how to make the command outputs nice to customers

Reply:
> I will need a refresher on what this is supposed to do

Its original intent was to buffer up errors and present them at once to users, where the prior implementation heavily depending on the embedded CLI pattern, there could be an error, say adding endpoint 2 out of of collection of 5 and that would show right away while work may be still happening.

Reply:
I will remove this for now since there is no more need for it